src/parser.ts

```diff
@@ -0,0 +1,50 @@
+import * as fs from 'fs';
+import { join } from 'path';
+import { ConfigOptions } from './types/Options';
+
+function parseLine(line: string): Record<string, string> {
+  const delimiter = '=';
+  const lineRegex = /^\s*[a-zA-Z_][a-zA-Z0-9_]*\s*="?'?.*'?"?$/g;
```

Comment:
Out of curiosity, where did this regex come from? Did you write it, or did you find it somewhere online?

Reply:
I wrote it!